stable/external-dns/templates/secret.yaml

```diff
@@ -7,10 +7,10 @@ metadata:
   name: {{ template "external-dns.fullname" . }}
 type: Opaque
 data:
-{{- if and .Values.aws.secretKey .Values.aws.accessKey }}
-  aws_access_key_id: {{ .Values.aws.accessKey | b64enc | quote }}
-  aws_secret_access_key: {{ .Values.aws.secretKey | b64enc | quote }}
-{{- end }}
+{{- if eq .Values.provider "aws" }}
+  credentials: {{include "external-dns.aws-credentials" . | b64enc | quote}}
```

Comment:
Space after `{{` and before `}}`, please.

Reply:
Done, I also added missing spaces in other templates.